src/lib/operators/export_binary.cpp

```diff
@@ -164,6 +174,56 @@ void ExportBinary::_write_chunk(const std::shared_ptr<const Table>& table, std::
   }
 }
 
+void ExportBinary::_write_partitioning_header(const std::shared_ptr<const Table>& table, std::ofstream& ofstream) {
+  const auto partition_schema = table->get_partition_schema();
```

Comment:
Please be concise in your usage of "partition" and "partitioning" (`partitioning_header` vs `partition_schema`). In my understanding, partitioning is the overall thing (criteria, number of partitions, schema, ...) and a partition is the single entity in that partitioning.